internal/artifact/artifact.go

```diff
@@ -84,6 +86,8 @@ func (t Type) String() string {
 		return "Brew Tap"
 	case GoFishRig:
 		return "GoFish Rig"
+	case KrewPlugin:
+		return "Krew Plugin"
```

Comment:
Not sure about the name. Because actually they are kubectl plugins... Tricky

Reply:
ended up naming it krew plugin manifest, which is the most specific name...